src/main/java/net/minecraftforge/event/entity/living/LivingEnchantmentLevelEvent.java

```diff
@@ -0,0 +1,45 @@
+package net.minecraftforge.event.entity.living;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.item.ItemStack;
+
+/**
+ * This event is fired when an enchantment level is being looked up for an entity.
+ * @author rubensworks
```

Comment:
This is not the documentation expected from events (from what I recall), [EntityEvent.java](https://github.com/MinecraftForge/MinecraftForge/blob/master/src/main/java/net/minecraftforge/event/entity/EntityEvent.java) contains the all perfect examples of how to doc events.

Also, the author tag is to never be used. Please remove it